api/src/main/java/org/openmrs/module/quarterlyreporting/db/daoimpl/QuarterlyReportingDAOImpl.java

```diff
@@ -3126,7 +3126,7 @@ public Boolean isLastRegimenProphy(Patient p) {
 		List<Integer> regimenDrugs = new ArrayList<Integer>();
 
 		for (RegimenComponent rc : components) {
-			if (!rc.getDrugOrder().getDiscontinued())
+			if (!rc.getDrugOrder().isActive())

```

Comment:
you are right , all reference I have  made active will be change as follows:
if (!rc.getDrugOrder().getDiscontinued()) will  be changed into if (rc.getDrugOrder().isActive())